x-pack/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -112,7 +112,9 @@ export const AuthenticationTable = connect(
   }
 )(AuthenticationTableComponent);
 
-const getAuthenticationColumns = (): [
+const getAuthenticationColumns = (
+  pageType: hostsModel.HostsType
```

Comment:
this is an unused parameter

Reply:
Oops! Good catch. Will fix. Thanks, Steph!